conn_test.go

```diff
@@ -97,6 +97,41 @@ func TestConn(t *testing.T) {
 		assert.Contains(t, err, "failed to wait for pong")
 	})
 
+	t.Run("pingPongReceived", func(t *testing.T) {
```

Comment:
If we ship the ability to modify payloads, could we also add a test that covers that?

Reply:
I removed the ability of modify payloads, but I have added the ability of deciding to send or not the pong response.
I have added tests to cover this.